frontend/src/components/KeyMeasureDiagnostic/ExpeModals/ExpeReservation.vue

```diff
@@ -0,0 +1,572 @@
+<template>
+  <div>
+    <div v-if="showSpinner" style="min-height: 290px; background-color: white">
+      <v-progress-circular indeterminate style="position: absolute; left: 50%; top: 50%"></v-progress-circular>
+    </div>
+    <v-card class="text-left" v-else-if="expe">
+      <v-card-title class="font-weight-black">
+        Vous êtes inscrit à l'expérimentation
+      </v-card-title>
+      <v-card-text>
+        Afin de pouvoir évaluer les effets de la solution de réservation sur la satisfaction des usagers et le taux de
+        fréquentation et de confirmer son potentiel en tant que levier de lutte contre le gaspillage alimentaire, nous
+        vous demandons de suivre et renseigner différents indicateurs nécessaires à l’évaluation du gaspillage
+        alimentaire, du taux de fréquentation et de la satisfaction des usagers à 3 étapes de l’expérimentation.
+      </v-card-text>
+      <v-card-text class="text-center">
+        <v-btn class="mr-4" color="primary" outlined>Télécharger le guide</v-btn>
+        <v-btn color="primary" text>Lire le décret de l'expérimentation</v-btn>
+      </v-card-text>
+      <v-divider></v-divider>
+      <v-card-text>
+        <v-form ref="form" v-model="formIsValid">
+          <!-- reservation system in place? -->
+          <v-checkbox v-model="expe.hasReservationSystem">
+            <template v-slot:label>
+              <span class="body-2 grey--text text--darken-2">
+                J'ai déjà mis en place une solution de réservation de repas
+              </span>
+            </template>
+          </v-checkbox>
+
+          <!-- reservation system date -->
+          <label v-if="expe.hasReservationSystem" class="body-2" for="date">
```

Comment:
Hmm le `for="date"` ne marche pas trop ici, je comprends pas trop. P-e on peut revisiter ça avec Anne-So et le réparer dans une deuxième PR